src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -34,8 +34,8 @@ internal sealed class AspNetCoreDiagnosticObserver : DiagnosticObserver
         private static readonly PropertyFetcher HttpRequestInStopHttpContextFetcher = new PropertyFetcher("HttpContext");
         private static readonly PropertyFetcher UnhandledExceptionHttpContextFetcher = new PropertyFetcher("HttpContext");
         private static readonly PropertyFetcher UnhandledExceptionExceptionFetcher = new PropertyFetcher("Exception");
-        private static readonly PropertyFetcher BeforeActionHttpContextFetcher = new PropertyFetcher("HttpContext");
-        private static readonly PropertyFetcher BeforeActionActionDescriptorFetcher = new PropertyFetcher("ActionDescriptor");
+        private static readonly PropertyFetcher BeforeActionHttpContextFetcher = new PropertyFetcher("httpContext");
+        private static readonly PropertyFetcher BeforeActionActionDescriptorFetcher = new PropertyFetcher("actionDescriptor");
```

Comment:
What's the deal here? Was this a bug? Did this hit privates instead of publics or something?

Reply:
The property names are actually lower case here. I just wanted them to match, but it doesn't make a real different since the code in case-insensitive. I noticed this because when I was making [this change](https://github.com/DataDog/dd-trace-dotnet/pull/642/files#diff-03ea678402f5d547eb5d427be0c0c583R35) I inadvertently make the code case-sensitive and the ASP.NET Core tests failed.

Reply:
[Here's an example](https://github.com/dotnet/aspnetcore/blob/474d03c071867dacb40dbdef993d6714b276f558/src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs#L229-L239) of how ASP.NET Core uses an anonymous classs to pass values into diagnostic listeners.